app/src/test/java/it/niedermann/owncloud/notes/android/fragment/SearchableBaseNoteFragmentTest.java

```diff
@@ -0,0 +1,106 @@
+package it.niedermann.owncloud.notes.android.fragment;
+
+import android.util.Log;
+
+import org.junit.Test;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.junit.Assert.*;
+
+public class SearchableBaseNoteFragmentTest {
+
+    @Test
+    public void testCountOccurrencesFixed() {
+        try {
+            Method method = SearchableBaseNoteFragment.class.getDeclaredMethod("countOccurrences", String.class, String.class);
+            method.setAccessible(true);
+
+            for (int count = 0; count <= 15; ++count) {
+                StringBuilder sb = new StringBuilder("Mike Chester Wang");
+                Random rand = new Random();
```

Comment:
I am not happy with this test.

If it fails, we won't even be able to reproduce it, since we don't have the test parameters in the fail output as far as i can see.

I would prefer to use a fixed value as test parameter (build one with a loop maybe), or at least add all test parameters to fail output.

Reply:
@WingsUpete, @Hui-Ouyang16. Could you please help me to fix this?

Reply:
Wait. What do you mean about "not be able to reproduce it"? Do you mean the random?

Reply:
I guess the reason for "not be able to reproduce" which you said is about the random. In fact, this variable is never used in the test file (I forgot to remove it). I have removed this variable and check again to make sure that there is no useless statement.